### PR TITLE
feat: commit re-commit text changes

### PR DESCRIPTION
**Pull request by radireddy**
ref: #MANAGER-7434

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7435 #MANAGER-7434
| License          | BSD 3-Clause

## Description

1. Remove US from currency symbol in US region. Like show "$20" instead of "US$20"
2. Show total saved amount in commitment upfront

### CODE REVIEW

File: packages/manager/modules/billing-components/src/components/commitment/commitment.controller.js [hunk 1]
```diff
@@ -104,9 +105,31 @@ export default class {
         (periodic.totalPrice.value / upfront.totalPrice.value - 1) * 100,
       );
       this.savings = periodic.getPriceDiff(upfront);
+      let totalSavings = this.savings.value;
+      totalSavings += this.model.duration.savings
```
Comment (varun257) [inline packages/manager/modules/billing-components/src/components/commitment/commitment.controller.js:109]: `totalSavings += this.model.duration.savings?.value || 0`

**Review by varun257 — DISMISSED** [review]

File: packages/manager/modules/billing-components/src/components/commitment/commitment.controller.js (hunk 1 above)
Comment (radireddy) [reply packages/manager/modules/billing-components/src/components/commitment/commitment.controller.js:109]: It just a another way of doing it. I don't see any performance improvements.

**Review by ganeshkumar1989 — DISMISSED** [review]

File: packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -27,5 +27,7 @@
   "billing_commitment_success_purchase": "Veuillez finaliser le paiement de votre <a href=\"{{url}}\" target=\"_blank\">bon de commande</a>.",
   "billing_commitment_error": "Une erreur est survenue.",
   "billing_commitment_summit": "Valider",
-  "billing_commitment_summary_description_end": "Après le {{ endDate }} votre service sera tacitement réengagé pour une durée similaire à votre dernier engagement."
+  "billing_commitment_summary_description_end": "Après le {{ endDate }} votre service sera tacitement réengagé pour une durée similaire à votre dernier engagement.",
+  "billing_commitment_savings": "Économisez un total de {{ price }}",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json:31]: may be better, to check:
```suggestion
  "billing_commitment_savings": "Économiser un total de {{ price }}",
```

**Review by mohammed-zahaf — DISMISSED** [review]

File: packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json (hunk 2 above)
Comment (antleblanc) [reply packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json:31]: @radireddy any update regarding @mohammed-zahaf's suggestion?
Do we have to move forward with actual diff or changes will be introduced?

Comment (mohammed-zahaf) [reply packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json:31]: Hello @antleblanc, i discuss on it with the reporter he will check with cntent.

Comment (mohammed-zahaf) [reply packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json:31]: So, i received reponse it must be :
  "billing_commitment_savings": "Économiser un total de {{ price }}",

File: packages/manager/modules/billing-components/src/components/commitment/commitment.html [hunk 3]
```diff
@@ -110,18 +110,27 @@
...
-                            <span
+                            <div
                                 data-translate="billing_commitment_payment_monthly"
                                 data-translate-values="{ price: engagement.pricing.monthlyPrice.text,  duration: engagement.durationInMonths}"
-                            ></span>
+                            ></div>
+                            <div
+                                data-ng-if="$ctrl.defaultPrice && $ctrl.model.duration.savings && $ctrl.model.duration.savings.value > 0"
+                                data-translate="billing_commitment_savings"
+                                data-translate-values="{ price: $ctrl.model.duration.savings.text}"
```
Comment (antleblanc) [inline packages/manager/modules/billing-components/src/components/commitment/commitment.html:132]:
```suggestion
                                data-translate-values="{ price: $ctrl.model.duration.savings.text }"
```

**Review by antleblanc — CHANGES_REQUESTED** [review]
According to the following commit `52b14eb`, you're referring to internal reference `MANAGER-74349`.
Unfortunately I cannot find it so could you please update it? Thanks!

**Review by varun257 — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Pull request merged**